packages/loader/test/index.test.js

```diff
@@ -138,6 +138,12 @@ webpack.mdx:1:22: Unexpected end of file in expression, expected a corresponding
     'should compile (preact)'
   )
 
+  assert.ok(
+    // @ts-expect-error Type definitions don’t define __source.
+    modPreact.default.default({}).__source,
```

Comment:
I'm not sure I follow this test case.
Does this property only exist in development mode?
If so, would we need a test to show it is missing in production mode?

Reply:
Yes, this property only exists in development mode. Currently a webpack build with mode `'none'` is created for React, and one with mode `'development'` for Preact. Preact attaches the `__source` property to the node, React does not (not even in development). I guess React tracks this somewhere internally.

Reply:
Interesting 🤔 
Would it make sense to also have a test in production mode, and verify the property is missing? Or is that assumed/implied by a test elsewhere?

Reply:
It’s also possible to read the compiled output and look for certain patterns. It’s not very elegant IMO, but it could work.

Reply:
I think there should be an explanation next to this test: I agree with Christian that reading this test code is confusing.

The `reactBuild` has `mode: 'none'`, so there it should use a “production” automatic jsx import, and here we should have a development import, perhaps we can `t.match` for those?

Reply:
I changed it to use `assert.match` now. It’s a bit annoying to find a propert string to match, as output changes significantly with the mode, but I found an ok regex to test against.